components/automate-chef-io/content/docs/troubleshooting.md

```diff
@@ -68,6 +68,14 @@ The following procedure will remove Chef Automate from your system,
 including all data. If you wish to preserve the data, make a backup
 before uninstalling.
 
+With the `chef-automate` CLI:
+
+```bash
+chef-automate uninstall
+```
+
+or
```

Comment:
You don't have to do this, but I would probably comment out this alternative if we'd rather people not do it but we want it available as an escape hatch to point someone to.